src/BrazeKit-dev.js

```diff
@@ -755,6 +755,10 @@ var constructor = function () {
                 }
             }
 
+            if (forwarderSettings.userIdentificationType === 'MPID' && mParticle.Identity.getCurrentuser().getMPID() != null) {
```

Comment:
I think you should err on the side of caution and check if both `Identity` and `getCurrentUser` are also not undefined.